TESTPLAN.md

```diff
@@ -2,13 +2,13 @@
 
 ## Automated Testing
 
-`npm test` verifies the telemetry payload as expected at firefox startup and add-on installation in a clean profile, then does **optimistic testing** of the *commonest path* though the study for a user
+`npm test` verifies the telemetry payload as expected at Firefox startup and add-on installation in a clean profile, then does **optimistic testing** of the *commonest path* though the study for a user
 
 - prove the notification bar ui opens
 - *clicking on the left-most button presented*.
 - verifying that sent Telemetry is correct.
 
-Code at [./test/functional_test.js].
+Code at [/test/functional_test.js](/test/functional_test.js).
```

Comment:
This link doesn't work; should be `[/test/functional_tests.js](/test/functional_tests.js)`.